pkgs/servers/documize-community/vendor.patch

```diff
@@ -0,0 +1,2392 @@
+diff --git a/vendor/github.com/microcosm-cc/bluemonday/.coveralls.yml b/vendor/github.com/microcosm-cc/bluemonday/.coveralls.yml
```

Comment:
This whole file is just a copy of another external library, right? Could it not be generated from a proper fetcher derivation so that it can be reasonably kept up to date?

Reply:
Feel free to send a PR, everything that was manually patched was pretty gnarly.